meta-mender-tegra/meta-mender-tegra-jetpack5/recipes-bsp/uefi/l4t-launcher-extlinux.bbappend

```diff
@@ -0,0 +1 @@
+UBOOT_EXTLINUX_FDT:jetson-orin-nano-devkit ?= "/boot/${DTBFILE}"
```

Comment:
Does anyone know what this is doing? Seems wrong that it would only be appended for the nano devkit.

Reply:
@irodriguez-veridas you had a Orin Nano right?  can you confirm if this is needed or not?